src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -848,6 +848,19 @@
     <Field type="float" contentType="hertz" name="switchRate" label="Switch Rate" description="Number of context switches per second" />
   </Event>
 
+  <Event name="CompilerQueueUtilization" category="Java Virtual Machine, Compiler" label="Compiler Queue Utilization" period="everyChunk">
+    <Field type="CompilerType" name="compiler" label="Compiler" />
+    <Field type="long" contentType="hertz" name="ingress" label="Ingress" description="Number of added requests per second"/>
```

Comment:
Could we use other terms then ingress and egress? Something that is more in general use,

Reply:
Thought about enqueue/dequeue, but went with added/removed so its consistent with the other fields